tests/StorageTestDefs.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * GPII Storage Integration Test Definitions
```

Comment:
These are really unit tests

Reply:
Bump

Reply:
Do you mean that we should rewrite them as unit test? (without the Integration tests infrastructure)